aries_vcx/src/protocols/issuance_v2/holder/states/credential_received.rs

```diff
@@ -0,0 +1,29 @@
+use messages::msg_fields::protocols::cred_issuance::v2::issue_credential::IssueCredentialV2;
+
+use crate::protocols::issuance_v2::formats::holder::HolderCredentialIssuanceFormat;
+
+pub struct CredentialReceived<T: HolderCredentialIssuanceFormat> {
+    #[allow(dead_code)] // `credential` may become used in future
+    pub(crate) credential: IssueCredentialV2,
```

Comment:
I am leaning toward removing this. You put a hard effort to eliminate redundancy and store minimal set of data inside `stored_credential_metadata`. But we store entire `issue-credential` message, it has privacy/security implications. 
In mobile setting, you would ideally want to store in the wallet not only all crypto data, but also all private data. If you store state machine in this state, you are leaking the private data in possibly un-encrypted storage.

Reply:
kk agreed